packages/swap/src/utils/swap.ts

```diff
@@ -46,16 +48,23 @@ export const estimateSwapDuration = async ({
   // assets are spendable by the user once the egress is included in a block
   const egressInclusionDuration = chainConstants[destChain].blockTimeSeconds;
 
-  const depositDuration =
-    depositInclusionDuration + depositWitnessDuration + depositWitnessSubmissionDuration;
-  const egressDuration = EGRESS_BROADCAST_SIGNING_DURATION + egressInclusionDuration;
+  const depositDuration = isExternal
+    ? depositInclusionDuration + depositWitnessDuration + depositWitnessSubmissionDuration
+    : 0;
+  const egressDuration = isExternal
+    ? EGRESS_BROADCAST_SIGNING_DURATION + egressInclusionDuration
+    : 0;
```

Comment:
we don't need to change this because below we don't add these properties to the object if `isExternal` is true